fairlearn/reductions/_moments/conditional_selection_rate.py

```diff
@@ -29,17 +33,38 @@ class ConditionalSelectionRate(ClassificationMoment):
 
     """
 
+    def __init__(self, ratio=1.0):
+        """Intialise with the ratio value."""
+        super(ConditionalSelectionRate, self).__init__()
```

Comment:
ise -> ize [for the consistency with the rest of our documentation that uses American spelling]